src/main/kotlin/domain/Winner.kt

```diff
@@ -0,0 +1,8 @@
+package domain
+
+class Winner(private val cars: List<Car>) {
+
+    private val maxPosition = cars.maxOf { it.position }
+
+    fun getWinner(): String = cars.filter { it.position == maxPosition }.joinToString { it.name }
```

Comment:
```suggestion
    fun getWinner(): String = cars.filter { it.position == maxPosition }.joinToString { it.name }
```

getWinner에 대해서 우승한 자동차 객체만 줘야한다고 생각해요. name을 주는 것은 view에 대한 영역이라고 생각이 드네요. 🤔